crate/plan.sh

```diff
@@ -1,68 +1,74 @@
 pkg_name=crate
 pkg_origin=core
-pkg_version="1.1.2"
+pkg_version="3.2.5"
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_license=('Apache-2.0')
 pkg_source="https://cdn.crate.io/downloads/releases/${pkg_name}-${pkg_version}.tar.gz"
-pkg_shasum="8f22b6531b3d1c8602a880779bbe09e5295ef0959a30aff0986575835aadc937"
+pkg_shasum="b40b066f53155bc13a2289396a9225f7246ad09c2cb95a4c02dc2d093758e56b"
+asc_checksum="4795f66c0a77016ee96c5e62d4c720b3a5ef56752e7aff5a58410bd58b93c132"
 pkg_build_deps=(core/gnupg)
-pkg_deps=(core/jre8 core/curl)
+pkg_deps=(
```

Comment:
I think we need to add `core/awk` to pkg_deps.  I'm getting 
```
hab pkg exec smacfarlane/crate crate
/hab/pkgs/smacfarlane/crate/3.2.5/20190403221229/crate/bin/crate.in.sh: line 77: awk: command not found
```
which is masked in the tests.